mlflow/models/model.py

```diff
@@ -777,6 +799,11 @@ def log(
             run_id: The run ID to associate with this model. If not provided,
                 a new run will be started.
             resources: {{ resources }}
+            auth_policy: Specifies the authentication policy for the model, which includes two key
```

Comment:
I did not mention "Downscoping" or "Definers Rights" in the comments, because in my opinion that is how Databricks is using this information. I dont think it would apply to OSS. Let me know if I should include that information as well